src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -81,16 +107,37 @@ export const Section: React.FC<SectionProps> = ({ venue }) => {
 
   if (!auditoriumSection) return <p>The section id is invalid</p>;
 
+  const userSeated =
+    typeof userWithId?.data?.[venue.id]?.row === "number" &&
+    typeof userWithId?.data?.[venue.id]?.row === "number";
+
   return (
     <div className="Section">
       <BackButton onClick={backToMain} locationName="overview" />
       <div className="Section__seats">
-        <IFrame
-          containerClassname="Section__iframe-overlay"
-          iframeClassname="Section__iframe"
-          iframeInlineStyles={iframeInlineStyles}
-          src={iframeUrl}
-        />
+        <div className="Section__iframe-overlay">
+          <IFrame
+            iframeClassname="Section__iframe"
+            iframeInlineStyles={iframeInlineStyles}
+            src={iframeUrl}
+          />
+          <div
+            style={reactionsContainerInlineStyles}
+            className="Section__reactions-container"
+          >
+            {userSeated ? (
+              <ReactionsBar
+                reactions={EmojiReactions}
+                venueId={venueId}
+                onClickLeaveSeat={leaveSeat}
+                isAudioEffectDisabled={isAudioEffectDisabled}
+                onClickMute={toggleMute}
+              />
+            ) : (
+              "Welcome! Click on an empty seat to claim it!"
+            )}
+          </div>
+        </div>
```

Comment:
Probably the name should be a bit different, not `iframe`. But Reactions should be part of the video

Reply:
> But Reactions should be part of the video

@mike-lvov What do you mean by this bit?

Reply:
Changes in https://github.com/sparkletown/sparkle/pull/1339/commits/66819ea73f97abf7d613bcc012802229098bee29

Reply:
though, it can be more beneficial to just review the latest version of that chunk of code

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1339/commits/d2a708581428aa087efe8271fe99a7fa1029bdbe
& before. Worth checking the latest version though